projects/packages/paypal-payments/src/block/featured-media.js

```diff
@@ -0,0 +1,69 @@
+import { BlockControls, BlockIcon, MediaPlaceholder, MediaUpload } from '@wordpress/block-editor';
+import { ToolbarGroup, ToolbarButton, ToolbarItem } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { get } from 'lodash';
+import EditButton from '../../shared/edit-button';
```

Comment:
You'd need to bring this in as well.

Reply:
I've brought the `edit-button` over in [7341e1a](https://github.com/Automattic/jetpack/pull/43413/commits/7341e1a4beed457c2d1d01bfcaba0ead695f87a4).